.github/actions/cypress-e2e-testing/action.yml

```diff
@@ -28,7 +28,7 @@ runs:
   steps:
     - uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # v4.0.0
       with:
-        version: 7
+        version: 8
```

Comment:
I'm not sure this will work :)
because pnpm lockfile is probably set to version 7